src/networktables/NetworkTable.cpp

```diff
@@ -20,11 +20,8 @@ unsigned int NetworkTable::s_port = NT_DEFAULT_PORT;
 void NetworkTable::Initialize() {
   if (s_running) Shutdown();
   if (s_client) {
-    std::vector<std::pair<StringRef, unsigned int>> servers;
-    servers.reserve(s_ip_addresses.size());
-    for (const auto& ip_address : s_ip_addresses)
-      servers.emplace_back(std::make_pair(ip_address, s_port));
-    nt::StartClient(servers);
+    nt::StartClient();
+    nt::SetServerFromDS(s_port);
```

Comment:
Should this be done automatically? It feels like it's something that should be enabled explicitly?